CHANGELOG.md

```diff
@@ -2,4 +2,7 @@
 
 ## Unreleased
 
+## 2.0.0 - First release
+
 - Initial version of Ruby buildpack in Rust (https://github.com/heroku/buildpacks-ruby/pull/93)
+- Version 2.0.0 for the first release is not a typo. There's an issue in pack where a builder with the same name and version number would reuse artifacts left on image from prior runs. Previously a `heroku/ruby` CNB was released with older versions. That buildpack was based on Ruby and needed to bootstrap a copy of Ruby meaning that future runs of any `heroku/ruby` CNB with the same versions would also contain that bootstrapped Ruby binary. This causes confusion and difficult to debug problems. The way to ensure no one would encounter that issue was to release using a version we know has not been used before. Version 2.0 was the first major version without a prior release of any `heroku/ruby` named CNB.
```

Comment:
The pack issue was fixed year ago fwiw:
https://github.com/buildpacks/pack/issues/1322#issuecomment-1038241038

Reply:
Updated. Also switched to the CHANGELOG header style in jvm and nodejs buildpacks.